screens/Marketplace.js

```diff
@@ -93,7 +93,7 @@ class MarketplaceBase extends Component {
   {
       const { type } = this.state; 
       const isActive = type===view.toLowerCase(); 
-      const displayTab = view==='Buying' ? 'Tutor' : 'Student';
+      const displayTab = view!='Buying' ? 'Tutor' : 'Student';
```

Comment:
This is still somehow flipped. Check final comment for more info.

Reply:
LOL i think Charles' change is correct cuz "I'm a Tutor", I sell. And as the screenshot shows, if "I'm a Student", I'll see posts that "come from a tutor". @SCAuFish @CharlesYu2000